webpack/JobWizard/steps/form/SelectField.js

```diff
@@ -34,9 +34,16 @@ export const SelectField = ({
         menuAppendTo={() => document.body}
         {...props}
       >
-        {options.map((option, index) => (
-          <SelectOption key={index} value={option} />
-        ))}
+        {options[0]?.id
```

Comment:
if one option has an id, does it mean the rest of the options will also have the same structure?
or should we map them anyway, and if they contain an id, change the value?
e.g:
```js
options.map((option, index) => (
              <SelectOption key={index} value={option.id ? option.id : option} />
           ))}
```